src/frequenz/sdk/config/_base_schema.py

```diff
@@ -0,0 +1,10 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Base schema for configuration classes."""
+
+from frequenz.quantities.experimental.marshmallow import QuantitySchema
+
+
+class BaseConfigSchema(QuantitySchema):
+    """A base schema for configuration classes."""
```

Comment:
Just wondering, why you create BaseConfigSchema? 
Can't you just derive QuantitySchema?

Reply:
> 1. Just wondering, why you create BaseConfigSchema?
> User can just use QuantitySchema

TL;DR: It is mostly to have a more future-proof API.

Maybe it is over-engineering for now, but my thought was that users might want to inherit from `BaseConfigSchema` and add their own custom fields. If in the future we have some other library providing a base schema like `QuantitySchema`, we can make `BaseConfigSchema` inherit from that too, and the user's code doesn't need to change, it will automatically get the new fields if they are inheriting from `BaseConfigSchema`.

If we use `QuantitySchema` (so users inherit from that), and then we want to ship a base schema with more fields than `QuantitySchema`, the users will either need to update their code to now inherit from `BaseConfigSchema`, or they won't get the new fields.

> 2. BaseConfigSchema is not used anywhere

Yeah, draft PR :-P Fixed in the new version.